widgets/blueprintCatalog/src/widget.tsx

```diff
@@ -99,9 +99,14 @@ Stage.defineWidget<
             widget.configuration.filter,
             widget.configuration.jsonPath
         );
-        return Promise.all([actions.doGetRepos(params), toolbox.getManager().doGet('/blueprints?_include=id', {})])
+        return Promise.all([
+            actions.doGetRepos(params),
+            toolbox.getManager().doGet('/blueprints?_include=id,state', {})
```

Comment:
Do we need to pass `{}`?

Will
```suggestion
            toolbox.getManager().doGet('/blueprints?_include=id,state')
```
work?

Even better would be to reuse `BlueprintActions.doGetBlueprints`.

Reply:
Can be removed

Reply:
to use `doGetBlueprints` need to add one more parameter there it's ok?

Reply:
nvm fixed